hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/codec/PauseWrapper.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2020-2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.services.store.contracts.precompile.codec;
+
+/*
+ * -
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ *
+ */
+
+import com.hederahashgraph.api.proto.java.TokenID;
+
+public record PauseWrapper(TokenID token) {
+
+    public static PauseWrapper pause(final TokenID token) {
```

Comment:
Remove this method since it's confusing. We don't want to actually `pause` anything but assign `TokenID` value. You can use directly the constructor generated from the `record` type.